test-end-to-end/webapp-cypress/cypress/support/commands.js

```diff
@@ -59,7 +59,15 @@ Cypress.Commands.add('ustadAddContentToLibrary',(contentPath,contentName) => {
   cy.get('input[id="content_title"]').click()
   cy.get('input[id="content_title"]').clear().type(contentName,{timeout: 2000})
   cy.get('#actionBarButton').click()
+})
 
+// Add a Folder to Library
+Cypress.Commands.add('ustadAddFolderToLibrary',(folderName) => {
```

Comment:
I would suggest you remove cy.contains("Library") from the command. This makes it impossible to use the command to create a subfolder.

The cy.contains("Library") click can be done in the test itself.